### DISCUSSION TITLE
Style: Use eto|woff|ttf font library file, compiled css gibberish

### DISCUSSION THREAD

**Discussion by leoWmz** [web-infra-dev/rsbuild#3448] [Q&A]
version:
`
"@rsbuild/core": "^1.0.1",
"@rsbuild/plugin-less": "^1.0.1-rc.5",
"@rsbuild/plugin-vue": "^1.0.1-rc.5"
`

![image](https://github.com/user-attachments/assets/ece4fef3-a873-4f2e-86ef-1693ba506869)

**Comment by chenjiahan** [comment]
See: https://github.com/web-infra-dev/rsbuild/issues/3328
